### PR TITLE
Dashboard card showing metrics requiring action.

### LINKED ISSUE #8938: Filter metrics requiring actions via the dashboard
Tasks:
- Add a "Metrics requiring action" card to the dashboard, showing the number of red, yellow, and white metrics in a little table.
- When clicking the card, only show the metrics requiring action via the existing setting.
- Make the card hideable via the settings.

### PR DESCRIPTION
**Pull request by fniessink**
Show a card in the dashboard with the number of metrics that require action (red, yellow, and white metrics). The card can be hidden via the Settings panel. Clicking the card or setting "Visible metrics" to "Metrics requiring actions" in the Settings panel hides metrics that do not require action.

Closes #8938.

### CODE REVIEW

File: components/frontend/src/dashboard/FilterCard.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.ui.card.filter_card .header {
+    margin-bottom: 0.3em;
+    overflow: hidden;
+    white-space: nowrap;
+}
+
+.ui.card.filter_card .header.blue {
+    color: #2185d0;
```
Comment (wkoot) [inline components/frontend/src/dashboard/FilterCard.css]: Do we not have one css file with all colour definitions?

File: components/frontend/src/utils.js [hunk 2]
```diff
@@ -254,7 +255,8 @@ sortWithLocaleCompare.propTypes = {
 
 function hideMetric(metric, metricsToHide, hiddenTags) {
     const hideBecauseNoActionNeeded =
-        metricsToHide === "no_action_needed" && ["target_met", "debt_target_met", "informative"].includes(metric.status)
+        metricsToHide === "no_action_required" &&
+        ["target_met", "debt_target_met", "informative"].includes(metric.status)
```
Comment (wkoot) [inline components/frontend/src/utils.js]: Perhaps it's handy to create a different const with the statuses that don't require action

File: components/frontend/src/dashboard/FilterCard.css (hunk 1 above)
Comment (fniessink) [reply components/frontend/src/dashboard/FilterCard.css]: No, but good suggestion. I'll add one.

Comment (fniessink) [reply components/frontend/src/dashboard/FilterCard.css]: I've added a few color variables to `App.css`, the rest we can do later.

File: components/frontend/src/utils.js (hunk 2 above)
Comment (fniessink) [reply components/frontend/src/utils.js]: Done.

**Pull request merged**